docs/en/observability/apm/manage-storage/ilm-how-to.asciidoc

```diff
@@ -124,17 +132,20 @@ Each policy includes a rollover and delete definition:
 
 |===
 
-The APM index lifecycle policies can be viewed in {kib}.
+// Is this right?
+// Does where they go in the UI change?
+The APM data stream lifecycle policies can be viewed in {kib}.
 Navigate to *{stack-manage-app}* / *Index Lifecycle Management*, and search for `apm`.
 
-TIP: Default {ilm-init} policies can change between minor versions.
+// Is this right?
+TIP: Default data stream lifecycle policies can change between minor versions.
```

Comment:
```suggestion
TIP: Default lifecycle policies can change between minor versions.
```
Maybe just reference that the lifecycle policies can change as a whole rather than specifically about DSL or ILM? Just an idea.